docs/copied-from-beats/dashboards.asciidoc

```diff
@@ -32,42 +32,45 @@ the _Beats Developer Guide_.
 ifndef::only-elasticsearch[]
 If you've configured the Logstash output, see
 <<load-dashboards-logstash>>.
-endif::[]
-
-ifdef::allplatforms[]
+endif::only-elasticsearch[]
 
 ifeval::["{requires-sudo}"=="yes"]
 
 include::../../libbeat/docs/shared-note-sudo.asciidoc[]
 
 endif::[]
 
+ifdef::deb_os,rpm_os[]
```

Comment:
With those changes, don't you have to specify `deb_os, rpm_os, win_os ..` in the `index.asciidoc` file?

Reply:
you are very right, good catch.